pkg/manager/manager.go

```diff
@@ -366,7 +366,23 @@ func (m *manager) AttachRepresentor(conf *types.PluginConf) error {
 		}
 	}()
 
-	if conf.Vlan != 0 {
+	// if VF has any VLAN config we should remove default vlan on port
+	// if VLAN 1 explicitly requested we should not remove it from the port
+	if conf.Vlan > 1 || len(conf.Trunk) > 0 {
+		if err = m.nLink.BridgeVlanDel(rep, uint16(1), true, true, false, true); err != nil {
```

Comment:
it hard to understand what is true and what is false. 
Maybe it better to create wrapper on the netlink api 
as you use true, true, false, true or false, false, false, true in the call.

Reply:
I agree, It is hard to read. I added simple wrapper functions with meaningful names for these actions.